tests/common/scripts/merging/test_merge_concat_without_separator.py

```diff
@@ -0,0 +1,50 @@
+import datetime
+
+from common.scripts.merging import merge_concat_without_separator
+
+
+def test_merge_concat_without_separator():
```

Comment:
I think it would be better to separate this into several tests (for each type)
And you can also use parametrization to improve readability and copy-paste errors: https://docs.pytest.org/en/6.2.x/parametrize.html